app/views/case_workers/admin/allocations/_allocation.html.haml

```diff
@@ -62,7 +62,7 @@
                 %td
                   %span.js-test-case-number
                     = link_to claim.case_number , case_workers_claim_path(claim)
-                    %div.unique-id-small= claim.unique_id
+                    %span.unique-id-small= claim.unique_id
```

Comment:
@cleargif changing to span makes the ID stay in the same line sometimes, instead of jumping to the next line.
This is not really an issue on production because the IDs are high, but will happen frequently on local and dev.

<img width="226" alt="screen shot 2016-07-13 at 17 00 27" src="https://cloud.githubusercontent.com/assets/687910/16810416/ab3e9a52-491b-11e6-99d1-7af19d913957.png">